toc/rfc/rfc-draft-remove-nonstandard-github-teams.md

```diff
@@ -0,0 +1,31 @@
+# Meta
+[meta]: #meta
+- Name: Remove Non-Standard Github Teams
+- Start Date: 2022-04-28
+- Author(s): @gerg
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: [https://github.com/cloudfoundry/community/pull/262](https://github.com/cloudfoundry/community/pull/262)
+
+
+## Summary
+
+Remove Github teams (and associated access) that are outside of the CFF
+governance model.
+
+## Problem
+
+There are currently many Github teams in the `cloudfoundry` Github organization
+that do not correlate with a CFF-recognized entity (working group, TOC, etc).
+These teams grant access to CFF-governed repositories, but are not beholden to
+the CFF's roles and other processes for governing access. In addition, the
+volume of Github teams makes it more difficult to manage team access via the
+Github UI and intermediary tools like
+[`cloudfoundry.yml`](https://github.com/cloudfoundry/community/blob/main/org/cloudfoundry.yml).
+
```

Comment:
* Acknowledge short-term access problems coming from the clean sweep of legacy teams.
* Point out some teams may be needed for access to repositories outside of CFF technical projects.
```suggestion

Nevertheless, controlling access to Cloud Foundry repositories only by teams defined by CFF technical governance is a substantial change from the previous system of access and team management and in the short term will cause unexpected loss of access to repositories for some contributors and automated development and release processes.

As per [RFC 0007](https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0007-repository-ownership.md), CFF staff also require access to many repositories in the Cloud Foundry GitHub organization that are outside the scope of project technical operation, such as those for CFF websites or the Cloud Foundry Certified Developer examinations.
```